playwright-tests/services/reef-knot.service.ts

```diff
@@ -0,0 +1,79 @@
+import { expect, test } from '@playwright/test';
+import { Page } from 'playwright';
+import { ReefKnotPage } from '../pages/reefKnot.page';
+import { WalletPage } from '@lidofinance/wallets-testing-wallets';
+import { TIMEOUT } from '../test-data/timeout.data';
+import { HDAccount, mnemonicToAccount } from 'viem/accounts';
+import { SdkService } from './sdk.service';
+
+export class ReefKnotService {
+  page: Page;
+  public walletPage: WalletPage;
+  reefKnotPage: ReefKnotPage;
+  seedPhrase: HDAccount;
+  sdkService: SdkService;
+
+  constructor(page: Page, walletPage: WalletPage, seedPhrase: string) {
+    this.page = page;
+    this.walletPage = walletPage;
+    this.reefKnotPage = new ReefKnotPage(this.page);
+    this.seedPhrase = mnemonicToAccount(seedPhrase);
+    this.sdkService = new SdkService(this.seedPhrase);
+  }
+
+  async isConnectedWallet() {
+    return test.step('Check wallet connection', async () => {
+      return this.reefKnotPage.header.isAccountSectionVisible();
```

Comment:
I believe from the Reef Knot perspective, it is also necessary to check the localStorage.

Reply:
Done